### PR TITLE
Prompt user for intproxy logs.

### PR DESCRIPTION
**Pull request by meowjesty**
- Issue: #2750

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -38,6 +38,12 @@ body:
       label: Relevant Logs
       description: If possible, reproduce the bug with the envrionment variable RUST_LOG set to 'trace', and attach the printed logs
       render: shell
+  - type: textarea
+    id: intproxy-logs
+    attributes:
+      label: Internal proxy logs
+      description: Please attach the relevant `mirrord-intproxy` logs. You may find them in your `/tmp` directory (default).
```
Comment (Razz4780) [inline .github/ISSUE_TEMPLATE/bug_report.yml:45]:
```suggestion
      description: Please attach the relevant `mirrord-intproxy` logs. You can find them in your `/tmp` directory (default).
```
Could you also add an example file name and mention that dst file can be configured?

File: mirrord/layer/src/error.rs [hunk 2]
```diff
@@ -227,7 +227,9 @@ impl From<HookError> for i64 {
             HookError::ProxyError(ref err) => {
                 graceful_exit!(
                     "Proxy error, connectivity issue or a bug. \n\
-                    You may report it to us on https://github.com/metalbear-co/mirrord/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml \n{err}"
+                    Please report it to us on https://github.com/metalbear-co/mirrord/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml \n\
+                    You may find the `mirrord-intproxy` logs in your `/tmp` directory (default).\n\
```
Comment (Razz4780) [inline mirrord/layer/src/error.rs:231]: I'd get layer config from `crate::setup()` and check if the destination was configured. If it was, I'd print exact absolute path instead

**Review by Razz4780 — APPROVED** [review]
Suggested a couple changes, but all optional.

Apart from the comments, I'd also change label and id of the field above - mirrord layer logs.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (meowjesty) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Changed and added an example.

File: mirrord/layer/src/error.rs (hunk 2 above)
Comment (meowjesty) [reply mirrord/layer/src/error.rs]: Good one, changed.

**Review by Razz4780 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -33,10 +33,19 @@ body:
       description: If your bug produces a backtrace, please include it here.
       render: shell
   - type: textarea
-    id: logs
+    id: layer-logs
     attributes:
-      label: Relevant Logs
-      description: If possible, reproduce the bug with the envrionment variable RUST_LOG set to 'trace', and attach the printed logs
+      label: mirrord layer logs
+      description: If possible, reproduce the bug with the envrionment variable RUST_LOG set to 'trace', and attach the printed logs.
```
Comment (aviramha) [inline .github/ISSUE_TEMPLATE/bug_report.yml:39]: typo in environment

Comment (meowjesty) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: I'm using archaic english :eyes: 


... fixed

**Pull request merged**